deeplake/core/vectorstore/vector_search/dataset/dataset.py

```diff
@@ -97,13 +100,22 @@ def load_dataset(
         verbose=False,
         **kwargs,
     )
-
     check_tensors(dataset)
 
     logger.warning(
         f"Deep Lake Dataset in {dataset_path} already exists, "
         f"loading from the storage"
     )
+
+    if runtime == {"tensor_db": True}:
```

Comment:
@adolkhan The runtime dictionary will evolve to contain other parameters. Therefore, we shouldn't do a full JSON match on the dicftionary. It should just check if the tensor_db key is True